backend/src/main/java/com/onlinebookstore/service/BookService.java

```diff
@@ -1,10 +1,14 @@
 package com.onlinebookstore.service;
 
+import com.onlinebookstore.dto.CreateBookRequest;
 import com.onlinebookstore.model.Book;
 import java.util.List;
+import org.springframework.web.bind.annotation.RequestBody;
 
 public interface BookService {
-    Book save(Book book);
+    Book save(@RequestBody CreateBookRequest requestDto);
```

Comment:
RequestBody is controller-level annotation
```suggestion
    Book save(CreateBookRequest requestDto);
```